### DISCUSSION TITLE
Adding code snippets to markdown

### DISCUSSION THREAD

**Discussion by kjaymiller** [kjaymiller/cookiecutter-relecloud#64] [Ideas]
From #62:

We should figure out consistent guidance for referencing code snippets from this project.

The snippets should be managed by the individual repos.

Here are the big questions that need to be answered.

- How do you get code lines
  - what if the lines are segmented
- How do you integrate them into markdown
  - Address formatting

**Comment by kjaymiller** [comment]
Symbex is an interesting solution - https://github.com/simonw/symbex

**Comment by pamelafox** [comment]
Learn.com has a way of referencing code snippets, by line number in a repo/branch. It formats them.

https://learn.microsoft.com/en-us/contribute/code-in-docs#in-repo-snippet-references
